src/wallet/wallet.cpp

```diff
@@ -2644,6 +2644,34 @@ bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nC
     return true;
 }
 
+OutputType CWallet::TransactionChangeType(const std::vector<CRecipient>& vecSend)
```

Comment:
`TransactionChangeType()` doesn't really need to be a class member function on `CWallet`, and could be a utility function in walletutil.cpp. In general, it'd be nice to start making the wallet class and wallet.cpp a bit smaller, but that's probably a tidy-up for another day.

Reply:
+1

Reply:
Agree, this should be moved at some point.